compiler/pipes/collect-required-and-classes.cpp

```diff
@@ -126,6 +127,46 @@ class CollectRequiredPass final : public FunctionPassBase {
     // we assume that if it returns an instance, it'll reference the required class in one way or another
   }
 
+  VertexPtr make_require_call(VertexPtr root, const std::string &name, bool once) {
+    auto file = require_file(name, true);
+    kphp_error_act (file, fmt_format("Cannot require [{}]\n", name), return root);
+    VertexPtr call = VertexAdaptor<op_func_call>::create();
+    call->set_string(file->main_func_name);
+    call->location = root->location;
+    if (once) {
+      VertexPtr cond = VertexAdaptor<op_log_not>::create(file->get_main_func_run_var());
+      call = VertexAdaptor<op_if>::create(cond, VertexAdaptor<op_seq>::create(call));
+      call->location = root->location;
+    }
+    return call;
+  }
+
+  // whether required file name refers to the composer-generated autoload file
+  static bool is_composer_autoload(const std::string &name) {
```

Comment:
Please, rename the parameter name to a more specific: file_name? class_name?

Reply:
Done.